app/static/profiles/book.json

```diff
@@ -245,12 +281,29 @@
             {
                 "id": "bfp:Instance:ElectronicBook",
                 "class": {
-                    "classLabel": "Electronic",
+                    "classLabel": "E-book",
                     "id": "http://bibfra.me/vocab/marc/Electronic",
-                    "instantiates": "http://bibfra.me/vocab/marc/Books",
```

Comment:
Switching to `instantiates` in the profile to better align with the vocab; please change.